CBLClient/TestCase.py

```diff
@@ -0,0 +1,243 @@
+from Client import Client
```

Comment:
This file is probably not needed anymore

Reply:
deleted